src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+/**
+ * Controller for Duke
+ */
+public class Duke {
+    static final String SAVE_FOLDER = "data/save.txt";
```

Comment:
Might want to consider changing the variable name. `SAVE_FOLDER` sounds a bit like a boolean type to me. Maybe try `SAVED_PATH` instead?